src/runtime/runtime_data.cs

```diff
@@ -96,7 +105,7 @@ internal static void RestoreRuntimeData()
 
         private static void RestoreRuntimeDataImpl()
         {
-            IntPtr capsule = PySys_GetObject("clr_data").DangerousGetAddress();
+            IntPtr capsule = PySys_GetObject("clr_data").DangerousGetAddressUnchecked();
```

Comment:
Same here, you don't need the address.

Reply:
I do need it. `PyCapsule_GetPointer` at line 113 needs an IntPtr. Sure I could add an overload for `PyCapsule_GetPointer`, but I think what we really need is an implicit operator.